frontend/cypress/tests/zoomView/zoomView_diffstat.cy.js

```diff
@@ -0,0 +1,89 @@
+describe('diffstat', () => {
+  it('should render container for contribution bars', () => {
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom .commit-message .stacked-bar-container')
+      .should('be.visible');
+  });
+
+  it('should render non-empty contribution bars for commits with changes', () => {
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom .commit-message')
+      .first()
+      .within(() => {
+        cy.get('.stacked-bar__contrib--bar')
+          .then((element) => {
+            expect(element.length).to.be.equal(2);
+            expect(element[0].style['background-color']).to.be.equal('limegreen');
+            expect(element[0].style.width).to.be.equal('0.1%');
+            expect(element[1].style['background-color']).to.be.equal('red');
+            expect(element[1].style.width).to.be.equal('0.1%');
```

Comment:
Would be good to add a short comment here regarding the assumptions that this test makes (like [this](https://github.com/reposense/RepoSense/blob/master/frontend/cypress/tests/zoomView/zoomView_rampChart.js)).